docs/release-notes-mtr/mtr_release_notes-1.0/master.adoc

```diff
@@ -0,0 +1,20 @@
+:{LC_PSN}:
+include::topics/templates/document-attributes.adoc[]
+:toc:
+:toclevels: 3
+:numbered:
+:imagesdir: topics/images
+:context: release-notes
+:release-notes:
+:_content-type: ASSEMBLY
+= {ReleaseNotesName}
+
+//Inclusive language statement
+include::topics/making-open-source-more-inclusive.adoc[]
+
+== Introduction
+{ProductName} provides an extensible and customizable rule-based tool that simplifies the migration and modernization of Java applications, such as migrating JBoss Enterprise Application Platform (EAP) 7 to 8 or migrating from any other application server towards EAP at scale. {ProductName} provides the same migration solution as provided in the Migration Toolkit for Applications 5 releases.
```

Comment:
Add ({ProductShortName}) after the first {ProductName}, then use {ProductShortName}.